### PR TITLE
CTOR-1154 : Refresh cloud-ibm-softlayer-api

### PR DESCRIPTION
**Pull request by lucie-dubrunfaut**
## Description

Refresh cloud-ibm-softlayer-api

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] 25.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-ibm-softlayer-api.md [hunk 1]
```diff
@@ -1,37 +1,364 @@
...
+l'utilisateur **centreon-engine** (`su - centreon-engine`). Vous pouvez tester
+que le connecteur arrive bien à superviser une ressource en utilisant une commande
+telle que celle-ci (remplacez les valeurs d'exemple par les vôtres) :
+
+```bash
+/usr/lib/centreon/plugins/centreon_ibm_softlayer_api.pl \
+	--plugin=cloud::ibm::softlayer::plugin \
+	--mode=events \
+	--api-username=''  \
+	--api-key=''  \
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-ibm-softlayer-api.md:223]: username and key

File: pp/integrations/plugin-packs/procedures/cloud-ibm-softlayer-api.md [hunk 2]
```diff
@@ -1,37 +1,369 @@
...
-### Centreon Plugin
+Your Centreon poller must have an access account for IBM SoftLayer.
+Ensure that you have an active IBM SoftLayer account with sufficient privileges to access resources via the API.
+The Centreon poller must be able to connect to the IBM SoftLayer API. The main API URL is: https://api.softlayer.com/rest/v3.1/
+If a firewall or proxy is configured, ensure that outbound connections to the SoftLayer API are allowed.
+You can test the connection to the API using the following curl command:
+```
+curl -u "USERNAME:API_KEY" https://api.softlayer.com/rest/v3.1/SoftLayer_Account/getAccount
+```
+For more information, refer to the [official documentation](https://sldn.softlayer.com/reference/softlayerapi/).
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-ibm-softlayer-api.md:72]:
```suggestion
* Your Centreon poller must have an access account for IBM SoftLayer.
* Ensure that you have an active IBM SoftLayer account with sufficient privileges to access resources via the API.
* The Centreon poller must be able to connect to the IBM SoftLayer API. The main API URL is: https://api.softlayer.com/rest/v3.1/
* If a firewall or proxy is configured, ensure that outbound connections to the SoftLayer API are allowed.
* You can test the connection to the API using the following curl command:
```
curl -u "USERNAME:API_KEY" https://api.softlayer.com/rest/v3.1/SoftLayer_Account/getAccount
```
* For more information, refer to the [Softlayer official documentation](https://sldn.softlayer.com/reference/softlayerapi/).
```

File: pp/integrations/plugin-packs/procedures/cloud-ibm-softlayer-api.md [hunk 3]
```diff
@@ -1,37 +1,369 @@
...
+Once the plugin is installed, log into your Centreon poller's CLI using the
+**centreon-engine** user account (`su - centreon-engine`). Test that the connector 
+is able to monitor a resource using a command like this one (replace the sample values by yours):
+
+```bash
+/usr/lib/centreon/plugins/centreon_ibm_softlayer_api.pl \
+	--plugin=cloud::ibm::softlayer::plugin \
+	--mode=events \
+	--api-username=''  \
+	--api-key=''  \
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-ibm-softlayer-api.md:228]: username and key

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-ibm-softlayer-api.md (hunk 1 above)
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-ibm-softlayer-api.md:223]:
```suggestion
	--api-username='XXXX'  \
	--api-key='XXXX'  \
```

File: pp/integrations/plugin-packs/procedures/cloud-ibm-softlayer-api.md (hunk 3 above)
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/cloud-ibm-softlayer-api.md:228]:
```suggestion
	--api-username='XXXX'  \
	--api-key='XXXX'  \
```

**Review by cg-tw — APPROVED** [review]

**Pull request merged**